heartbeat/monitors/util.go

```diff
@@ -342,3 +448,55 @@ func filterIPs(ips []net.IP, filt func(net.IP) bool) []net.IP {
 	}
 	return out
 }
+
+// MakeJobSetting creates a new JobSettings structure without any global event fields.
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method JobSettings.WithFields should have comment or be unexported